src/components/MovieDetails/MovieDetailsBody/MovieDetailsBody.tsx

```diff
@@ -0,0 +1,60 @@
+import { useAppSelector } from "../../../redux/hooks";
+import { selectMovieId } from "../../../redux/movieSlice";
+import { useGetMovieQuery } from "../../../redux/apiSlice";
+import { Spinner } from "../../common/Spinner/Spinner";
+
+import styles from "./movieDetailsBody.module.css";
+
+function MovieDetailsBody() {
+  const movieId = useAppSelector(selectMovieId);
+  const { data: movie, isFetching, isSuccess } = useGetMovieQuery(movieId);
+  const hour = Math.trunc(Number(movie?.runtime) / 60);
+  const minute = Number(movie?.runtime) % 60;
```

Comment:
It would be better to have this in separate utils file

Is that looks better?
```javascript
export const getRuntime = (time: number) => {
  const hours = Math.floor(time / 60);
  return `${hours}h ${time - hours * 60}m`;
};
```